docs/updating_schemas.md

```diff
@@ -215,6 +219,7 @@ Exception: When making substantial changes to an existing schema, even if these
 
 * **NOTE:** You should update the schema version when making substantial changes to an existing schema even if these changes do not cause existing objects using this schema to fail validation.
 The new schema version number in such cases helps submitters and users realize that a substantial change has been made to the schema and they may need to update their scripts accordingly.
+If you are updating the schema version, you should also update the appropriate change log markdown file with documentation of the new schema version, in **schemas/changelogs** directory.
```

Comment:
We have to update the changelog EVEN if the schema version number has not been bumped up. It appears as "minor changes since schema version xyz" see changelogs in encoded for an example.

Reply:
Done